README.md

```diff
@@ -9,6 +9,10 @@ SERPS - CURL Client
 
 This is a CURL Client implementation for [SERPS](https://github.com/serp-spider/serps)
 
+Requirements
+-------
+cURL >= 7.48.0
+
```

Comment:
You should update this file instead https://github.com/serp-spider/documentation/blob/master/docs/http-client/curl.md